_includes/footer.html

```diff
@@ -97,17 +97,17 @@ <h6 class="title">Latest Updates</h6>
           </li>
           <li>
             <a href="https://gitter.im/fossasia/fossasia" target="_blank" rel="noopener noreferrer">
-              <i class="fab fa-gitter"></i>
+              <i class="fa fa-comment"></i>
             </a>
           </li>
           <li>
             <a href="https://www.weibo.com/u/5784920339" target="_blank" rel="noopener noreferrer">
-              <i class="fab fa-weibo"></i>
+              <i class="fa fa-weibo"></i>
```

Comment:
Why is this changed?

Reply:
Again, I have used fontawesome 4.7 instead of fontawesome 5 since the later was not giving the expected icons

Reply:
Please use frontawesome 5 only

Reply:
The icon is correct and works for weibo, hence these changes are justified 
Also font awesome 5 icon for weibo is not working